public/_includes/documentation-nav.ejs

```diff
@@ -15,7 +16,7 @@ function endsWith(str, suffix) {
     <% /* For each article in documentation/_data.json */ 
        for(var item1 in public.documentation._data) { 
          var article1 = public.documentation._data[item1];
-
+         if ( article1.language === "en" ) {
```

Comment:
For navigation, is it possible to use one set of files instead of 2 duplicate files?  The only difference between this and the _zh one is that everywhere it says "if (article1.language === "zh")...."   For the content, it makes sense to have duplicate files because they are translations, but for the navigation and rules such as this, can we design it to have one set of files that is language aware?